Kiwi/PagedListController.swift

```diff
@@ -127,15 +127,7 @@ open class PagedListController<R: URIdentifiable>: NSObject {
 
             if let error = error {
                 page?.state = .error(error)
-                switch self.state {
-                case .loaded:
-                    //TODO: A page download has failed with previous pages already loaded
-                    //for now we don't wont the listController state set to error
-                    //Anyway forward a message to UI is needed
-                    break
-                default:
-                    self.setState(.error(error))
-                }
+                self.setState(.error(error))
```

Comment:
Estamos seguros de querer cambiar esto??

Reply:
En las dos apps que hemos revisado, se usa desde hace tiempo la nueva version (ignorando el estado del interactor si hay error). Yo lo dejaria como esta poniendo un comentario para que en un futuro decidamos que hacemos en este caso. Hagamos que funcione AS IS y luego lo mejoramos.

Reply:
@filipetamota puedes añadir el comentario de vuelta?